lib/utils/fs_test.go

```diff
@@ -347,3 +351,12 @@ func TestRemoveSecure(t *testing.T) {
 	_, err = os.Stat(tempFile)
 	require.True(t, os.IsNotExist(err), "File should be removed")
 }
+
+func TestRemoveSecureSymlink(t *testing.T) {
```

Comment:
Suggestion: distinguish scenario from function being tested.

```suggestion
func TestRemoveSecure_symlink(t *testing.T) {
```